src/main/java/com/example/webapp/Model/Department.java

```diff
@@ -14,18 +16,19 @@
 import javax.persistence.Table;
 import javax.validation.constraints.NotBlank;
 import javax.validation.constraints.NotEmpty;
-import java.util.Objects;
+import java.util.Set;
 
 @Entity
 @Table
 @Getter
 @Setter
```

Comment:
Getter Setter NoArgsConstructor AllArgsConstructor  can be substituted with data annotation

Reply:
Changed only Getter and Setter with Data.